fetch.bs

```diff
@@ -4754,13 +5012,21 @@ optional boolean <var>forceNewConnection</var> (default false), run these steps:
    `<code>Transfer-Encoding</code>`/`<code>chunked</code>` to <var>request</var>'s
    <a for=request>header list</a>.
 
+   <li>Set <var>timingInfo</var>'s <a for="fetch timing info">request start time</a> to the
+   <a for=/>unsafe shared current time</a>.
+
    <li>
     <p>Set <var>response</var> to the result of making an HTTP request over <var>connection</var>
     using <var>request</var> with the following caveats:
 
     <ul>
      <li><p>Follow the relevant requirements from HTTP. [[!HTTP]] [[!HTTP-SEMANTICS]] [[!HTTP-COND]] [[!HTTP-CACHING]] [[!HTTP-AUTH]]
 
+     <li><p>Set <var>timingInfo</var>'s <a for="fetch timing info">response start time</a> to
+     the <a for=/>unsafe shared current time</a>immediately after the user agent's HTTP parser
+     receives the first byte of the response (e.g. frame header bytes for HTTP/2, or response
+     status line for HTTP/1.x).
+
      <li><p>Wait until all the <a for=/>headers</a> are transmitted.
 
      <li>
```

Comment:
Comment on line 5034: What should we do with 103s? Do we want to report them as responseStart?

Might be best to add a TOOO here, open an issue on Resource Timing, or both..
https://tools.ietf.org/html/rfc8297

Reply:
It says "Any responses whose status is in the range 100 to 199, inclusive, and is not 101, are to be ignored", I don't think it needs a TODO

Reply:
Yeah, I'm not sure we want to ignore 103s (nor am I sure we want to mark responseStart when they arrived). I can just open an issue for the WebPerfWG to discuss this, and then we can PR whatever change we'd land on.

Reply:
Digging further, we explicitly have a test that contradicts this: https://github.com/web-platform-tests/wpt/blob/master/resource-timing/resource-timing-level1.js#L238

Chromium and Mozilla agree on this: https://wpt.fyi/results/resource-timing/resource-timing-level1.sub.html?label=experimental&label=master&aligned (with Safari failing)

Reply:
What do you suggest? This is copied from the current Resource-Timing spec.

Reply:
Re-reading the algorithm, it seems to first set the response start time, and only then saying the responses are to be ignored, which makes sense.
Might be worthwhile to add a "regardless of status code" to line 5028, to clarify that response start is registered there even for 1XX responses.

Reply:
Done